presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewQueryOptimizer.java

```diff
@@ -83,6 +85,10 @@
     private final Table materializedView;
     private final Query materializedViewQuery;
     private final LogicalRowExpressions logicalRowExpressions;
+    private final ImmutableSet<QualifiedName> supportedFunctionCalls = ImmutableSet.of(
```

Comment:
Keep the variable as set only instead of ImmutableSet?

Reply:
@jainxrohit Since this set is fixed, shouldn't we keep this as ImmutableSet (which is better than set I persume)?

Reply:
private final Set<QualifiedName> supportedFunctionCalls = ImmutableSet.of(